hedera-node/hedera-consensus-service-impl/src/test/java/com/hedera/node/app/service/consensus/impl/test/handlers/ConsensusCreateTopicHandlerTest.java

```diff
@@ -217,17 +210,18 @@ void createAddsPayerAsSubmitter() {
     @DisplayName("Fails if payer is not found")
     void createFailsWhenPayerNotFound() {
         // given:
-        given(keyFinder.getKey((AccountID) any()))
+        final var missing = com.hedera.hapi.node.base.AccountID.newBuilder().accountNum(1234).build();
+        given(keyFinder.getKey(missing))
                 .willReturn(KeyOrLookupFailureReason.withFailureReason(
-                        ResponseCodeEnum.ACCOUNT_ID_DOES_NOT_EXIST)); // Any error response code
+                        com.hedera.hapi.node.base.ResponseCodeEnum.ACCOUNT_ID_DOES_NOT_EXIST)); // Any error response code
         final var inputTxn = newCreateTxn(null, null, false);
 
         // when:
-        final var context = new PreHandleContext(keyFinder, inputTxn, IdUtils.asAccount("0.0.1234"));
+        final var context = new PreHandleContext(keyFinder, inputTxn, missing);
         subject.preHandle(context);
 
         // then:
-        assertThat(context.getStatus()).isEqualTo(ResponseCodeEnum.INVALID_PAYER_ACCOUNT_ID);
+        assertThat(context.getStatus()).isEqualTo(com.hedera.hapi.node.base.ResponseCodeEnum.INVALID_PAYER_ACCOUNT_ID);
```

Comment:
please check all the places where you added the package since mostly at all places the old classes have been removed and a regular import can be used